utils/artemis_utils/__init__.py

```diff
@@ -4,6 +4,7 @@
 import logging.handlers
```

Comment:
maybe it's better for these to be in a different file and not init?

Reply:
@slowr you are right, I will do this after the refactoring is complete. Currently working on the prefixtree, and I also need to re-check a bit the conf/start logic for the mandatory services (e.g., configuration, database).

Reply:
Done (not the refactoring, the move), moved to rest_util. Resolving this and continuing refactoring.